dynawo/sources/Models/CPP/Controls/Voltage/ModelVoltageMeasurementsUtilities/DYNModelVoltageMeasurementsUtilities.cpp

```diff
@@ -187,6 +187,9 @@ ModelVoltageMeasurementsUtilities::evalJCalculatedVarI(unsigned iCalculatedVar,
       }
       break;
     }
+    case minIValIdx_:
+    case maxIValIdx_:
```

Comment:
Is this the expected behaviour? Can this create any trouble in the process? I suspect this is the appropriate behaviour since infinitely small  variations in the input wouldn't change the output of the indices of max/min location (almost everywhere anyway) . But there might be some sort of issues in the computations. @cossartque ?